### PR TITLE
add example docker-compose and k8s files.

### PR DESCRIPTION
**Pull request by mmou**

### CODE REVIEW

File: docs/docker-compose-ca.yml.example [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Example docker-compose file for the SSH CA.
+version: '3.7'
+services:
+  kbsshca:
+    image: ca:latest
+    container_name: kbsshca
+    #command: ["./docker/entrypoint-server.sh"]
+    #command: ["./docker/entrypoint-generate.sh"]
+    command: ["sh", "-c", "sleep 1000"]
```
Comment (ddworken) [inline docs/docker-compose-ca.yml.example]: I think your idea here is to have someone manually comment out the correct line to use as the entrypoint, right? If so, fine by me but probably just add a comment to this file here explaining this.

File: docs/docker-compose-ca.yml.example [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    environment:
+      TEAMS: 'list,of,teams'
+      KEYBASE_USERNAME: 'yourusername'
+      KEYBASE_PAPERKEY: 'your paper key' # ideally, add this as a docker secret, and not in plaintext here
+      FORCE_WRITE: 'false'
+    volumes: # mounting the entrypoint scripts for easy iterative development.
```
Comment (ddworken) [inline docs/docker-compose-ca.yml.example]: Should this be in the version we commit?

Comment (mmou) [reply docs/docker-compose-ca.yml.example]: removed

File: docs/docker-compose-ca.yml.example (hunk 1 above)
Comment (mmou) [reply docs/docker-compose-ca.yml.example]: commented

**Pull request merged**